rero/rero-ils
# ebooks: display holdings

How to test:
- `./script/setup`
- Check in detail view document if the link to ebook is OK

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>
Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
 Co-Authored-by: Aly Badr <aly.badr@rero.ch>
